models/buildables/telenode/telenode.iqe.cfg

```diff
@@ -1,4 +1,4 @@
 output models/buildables/telenode/telenode.iqm
 rotate 0 -180 0
-scale .7569
+scale 0.65
```

Comment:
.7569 * 0.9 would be 0.68

Reply:
Yes but this was manually adjusted to be even better as seen get the screenshot above. The `0.9` scaling value was likely a rough approximation but it could be improved.

Reply:
With 0.68 the player's foot was a a bit within the telenode when standing over in screenshot  from https://github.com/UnvanquishedAssets/res-buildables_src.dpkdir/pull/20#issuecomment-2392504021

This was adjusted as seen in screenshot from https://github.com/UnvanquishedAssets/res-buildables_src.dpkdir/pull/20#issuecomment-2392506191